stan-native/src/main/java/org/ballerinalang/nats/connection/NatsStreamingConnection.java

```diff
@@ -45,7 +45,7 @@
 
     public static StreamingConnection createConnection(BObject streamingClientObject, String url,
                                                        String clusterId, Object clientIdNillable,
-                                                       Object streamingConfig)
+                                                       BMap streamingConfig)
```

Comment:
Shall we avoid using Generics here?
```suggestion
                                                       BMap<BString, Object> streamingConfig)
```